extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -349,9 +349,9 @@ export class BookTocManager implements IBookTocManager {
 			this.cleanUp(path.dirname(section.book.contentPath));
 		}
 
-		if (book.version === BookVersion.v1) {
+		if (book.book.version === BookVersion.v1) {
```

Comment:
Would suggest renaming the book param to `bookItem` to make it clearer what it is - book.book doesn't initially seem to make a lot of sense